src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -22,6 +23,7 @@ public class ModelManager implements Model {
     private final AddressBook addressBook;
     private final UserPrefs userPrefs;
     private final FilteredList<Project> filteredProjects;
+    private final ArrayList<Project> targetProject = new ArrayList<>();
```

Comment:
Good oop abstraction but the use of arraylist is quite awkward will try to see what I can do about it.